string type
string id
string settings
---
bool success
string message
